Automattic/wp-calypso
# Show store link in sidenav by default if user has a business plan

#### Changes proposed in this Pull Request

The current approach uses the `canCurrentUserUseStore` reducer to determine whether to show the store link in the sidenav. This PR changes the logic to check whether the user has a business or eCommerce plan instead.  

#### Testing instructions



* Sign up for a business plan
* Verify that the store sidenav link shows up
* Go through the store setup process
* Switch to a non-business / non-ecommerce plan and verify that the store sidenav link is hidden

Fixes #33341
